apps/backoffice-v2/src/pages/TransactionMonitoringAlerts/components/AlertsTable/columns.tsx

```diff
@@ -84,6 +86,39 @@ export const columns = [
     },
     header: 'Subject',
   }),
+  columnHelper.accessor('subject.type', {
+    cell: info => {
+      const subjectType = info.getValue();
+
+      return (
+        <TextWithNAFallback className={`max-w-[12ch]`}>{titleCase(subjectType)}</TextWithNAFallback>
+      );
+    },
+    header: 'Subject Type',
+  }),
+  columnHelper.accessor('subject.correlationId', {
+    cell: info => {
+      // eslint-disable-next-line react-hooks/rules-of-hooks -- ESLint doesn't like `cell` not being `Cell`.
+      const { ref, styles } = useEllipsesWithTitle<HTMLSpanElement>();
```

Comment:
Consider using the `truncate` CSS class instead.

Reply:
This is actually what makes the tooltip exist so I will leave it for now. We can review in the future.